packages/components-js/src/examples/pin-code-example-controlled.html

```diff
@@ -0,0 +1,15 @@
+<p-pin-code label="Some Label"></p-pin-code>
+
+<p-text class="value">Current value:</p-text>
+<p-text class="complete">Completely filled: false</p-text>
+
+<script>
+  const pinCode = document.querySelector('p-pin-code');
+  const debugElement1 = document.querySelector('.value');
+  const debugElement2 = document.querySelector('.complete');
+
+  pinCode.addEventListener('update', (e) => {
+    debugElement1.innerText = `Current value: ${pinCode.value}`;
```

Comment:
```suggestion
    pinCode.value` = e.target.value;
    debugElement1.innerText = `Current value: ${pinCode.value}`;
```
Update the value prop with e.target.value